pkg/collectors/route.go

```diff
@@ -0,0 +1,133 @@
+package collectors
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/kube-state-metrics/pkg/metric"
+	"k8s.io/kube-state-metrics/pkg/version"
+
+	"github.com/golang/glog"
+	"github.com/openshift/api/route/v1"
+	routeclient "github.com/openshift/client-go/route/clientset/versioned"
+)
+
+var (
+	descRouteLabelsName          = "openshift_route_labels"
+	descRouteLabelsHelp          = "Kubernetes labels converted to Prometheus labels."
+	descRouteLabelsDefaultLabels = []string{"namespace", "route_name"}
+
+	routeMetricFamilies = []metric.FamilyGenerator{
+		metric.FamilyGenerator{
+			Name: "openshift_route_created",
+			Type: metric.MetricTypeGauge,
+			Help: "Unix creation timestamp",
+			GenerateFunc: wrapRouteFunc(func(d *v1.Route) metric.Family {
+				f := metric.Family{}
+
+				if !d.CreationTimestamp.IsZero() {
+					f.Metrics = append(f.Metrics, &metric.Metric{
+						Value: float64(d.CreationTimestamp.Unix()),
+					})
+				}
+
+				return f
+			}),
+		},
+		metric.FamilyGenerator{
+			Name: "openshift_route_info",
```

Comment:
Could you split this into to metrics ? "openshift_route_info" that includes the info in the "spec", and "openshift_route_status" that describe the "status" part? WTDY?